dockerfiles/Dockerfile.cuda

```diff
@@ -14,8 +14,6 @@ RUN apt-get update && apt-get install -y --no-install-recommends python3-dev ca-
 
 RUN cd /code && /bin/bash ./build.sh --skip_submodule_sync --cuda_home /usr/local/cuda --cudnn_home /usr/lib/x86_64-linux-gnu/ --use_cuda --config Release --build_wheel --update --build --parallel --cmake_extra_defines ONNXRUNTIME_VERSION=$(cat ./VERSION_NUMBER) 'CMAKE_CUDA_ARCHITECTURES=30;37;50;52;60;70'
 
-FROM nvidia/cuda:10.2-cudnn8-runtime
```

Comment:
But I still prefer to use a two-phrase approach(keep the FROM line) so that we don't include the other unnecessary dependencies into the final images. For example:  gcc.

And the "nvidia/cuda:10.2-cudnn8-devel" base image is 2.5x larger(5GB vs 2GB) than what we are using now.  I wonder if it is worth.

Reply:
I actually didn't consider the size of the resulting image. I guess an alternative would be to copy the required .so files from  base image to resulting image. Let me know if anyone is working on it, else I can try that approach.